libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/location/NavigationLocationProvider.kt

```diff
@@ -176,4 +216,14 @@ class NavigationLocationProvider : LocationProvider {
             clientOptions?.also { apply(it) }
         }
     }
+
+    companion object {
+        /**
+         * Get the registered instance or create one and register it to [MapboxNavigationApp].
+         */
+        @JvmStatic
+        fun getInstance(): NavigationLocationProvider = MapboxNavigationApp
```

Comment:
```suggestion
        fun getRegisteredInstance(): NavigationLocationProvider = MapboxNavigationApp
```

Reply:
Should also update `MapboxAudioGuidance` https://github.com/mapbox/mapbox-navigation-android/blob/6e70c9ba3bef09979f8834401c33378c784d4b6d/libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/MapboxAudioGuidance.kt#L212-L214

Reply:
Updating here https://github.com/mapbox/mapbox-navigation-android/pull/6445